CHANGELOG.md

```diff
@@ -82,14 +82,6 @@ and a dynamic pool for those that are larger.
 - Fix CodeCov reporting by explicitly downloading older version of upload
   script.
 
-## [0.3.5] - 2019-06-11
```

Comment:
This 0.3.5 block needs to be put back.

Reply:
good catch. I used `--strategy=ours`. Will fix now.